setup.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+with open('requirements.txt') as f:
+    requirements = f.read().splitlines()
+    requirements = [req.replace('==', '>=') for req in requirements]
+
+
+setuptools.setup(
+    name='pollination-honeybee-vtk',
+    author='ladybug-tools',
+    author_email='info@ladybug.tools',
+    maintainer='Devang, ladybug-tools',
+    maintainer_email='devang@ladybug.tools, info@ladybug.tools',
+    packages=setuptools.find_namespace_packages(
+        include=['pollination.*'], exclude=['tests', '.github']
+    ),
+    install_requires=requirements,
+    use_scm_version=True,
+    setup_requires=['setuptools_scm'],
+    url='https://github.com/pollination/honeybee-vtk',
+    project_urls={
+        'icon': 'https://raw.githubusercontent.com/ladybug-tools/artwork/master/icons_bugs/grasshopper_tabs/Honeybee.png',
```

Comment:
Should we create a new icon for Honeybee VTK?

Reply:
I added the vtk icon

Reply:
This needs to be a url. I see that you are using a local path in the repository. That will not work. You can use this url instead: https://raw.githubusercontent.com/pollination/honeybee-vtk/master/assets/logo/vtk.png

Reply:
I fixed this. Thanks!